static/src/javascripts-legacy/projects/common/modules/tailor/tailor.js

```diff
@@ -1,34 +1,95 @@
 define([
-    'lib/fetch-json'
+    'Promise',
+    'lib/fetch-json',
+    'lib/cookies',
+    'lib/storage',
+    'lib/report-error'
 ], function(
-    fetchJson
+    Promise,
+    fetchJson,
+    cookies,
+    storage,
+    reportError
 ) {
-    function getEmail(browserId) {
-        return fetchJson('https://tailor.guardianapis.com/email/' + browserId +'?emailIds=39,3322,3039,1950,38,3698', {
-            method: 'get'
-        });
+
+    var URLS = {
+        suggestions: 'https://tailor.guardianapis.com/suggestions?browserId='
```

Comment:
Sure, so if it's off we return an empty object (unless we have data in local storage)...?